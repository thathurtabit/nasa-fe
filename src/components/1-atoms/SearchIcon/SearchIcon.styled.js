import styled from 'styled-components';

const SearchIconStyled = styled.svg`
  position: relative;
  fill: ${props => props.theme.colors.copy};
`;

export default SearchIconStyled;
