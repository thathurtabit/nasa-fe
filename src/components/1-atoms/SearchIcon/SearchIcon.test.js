import React from 'react';
import { shallowToJson } from 'enzyme-to-json';
import 'jest-styled-components';
import {
  shallowWithTheme,
  mountWithTheme,
} from '../../../utils/test_config/testHelpers';

import SearchIcon from './SearchIcon';

const title = 'test title';

describe('SearchIcon', () => {
  it('should render correctly', () => {
    const element = shallowWithTheme(
      <SearchIcon width="20" height="20" title={title} />
    );
    expect(shallowToJson(element)).toMatchSnapshot();
  });

  it('should return title attr correctly', () => {
    const element = mountWithTheme(
      <SearchIcon width="20" height="20" title={title} />
    );
    expect(element.find('title').text()).toEqual(title);
  });
});
