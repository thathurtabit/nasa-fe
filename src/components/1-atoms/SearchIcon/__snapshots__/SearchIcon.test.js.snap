// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchIcon should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
